endoreg_db/models/data_file/base_classes/abstract_video.py

```
@@ -1,18 +1,16 @@
 import shutil
 import os
-import subprocess
 from typing import Optional, List, TYPE_CHECKING, Union
-from django.db import models, transaction
 from icecream import ic
-from tqdm import tqdm
 from endoreg_db.utils.hashs import get_video_hash
 from endoreg_db.utils.file_operations import get_uuid_filename
 from endoreg_db.utils.ocr import extract_text_from_rois
-
+from pathlib import Path
```

Ruff (0.8.2)

Redefinition of unused `Path` from line 4

Remove definition: `Path`

(F811)